config/local/discoverable-client.yml

```diff
@@ -5,18 +5,14 @@ spring:
         ansi:
             enabled: always
 
+# The `apiml` node contains API Mediation Layer specific configuration
 apiml:
-    security:
-        ssl:
-            ciphers: TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256,TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA256,TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA384
-    # The `apiml` node contains API Mediation Layer specific configuration
     service:
         # The `apiml.service` node contains information required by any APIML service
-        id: discoverableclient  # Service ID of this service
```

Comment:
Why use serviceId under service? Just id is not enought?

Reply:
The config is deserialized into ApiMediationServiceConfig class. It has serviceId attribute, not id. There is no need to have serviceId: ${apiml.service.id} in internal file (which matters for config) and then apiml.service.id=.... in externalized. It is just confusing